cumulusci/core/tests/test_flowrunner.py

```diff
@@ -131,6 +131,21 @@ def test_init(self):
         self.assertEqual(len(flow.steps), 1)
         self.assertEqual(hasattr(flow, "logger"), True)
 
+    def test_show_summary(self):
+        flow_config = FlowConfig(
+            {
+                "description": "test description",
+                "steps": {"1": {"flow": "nested_flow_2"}},
+            }
+        )
+        flow = FlowCoordinator(self.project_config, flow_config, name="test_flow")
+        flow.show_summary()
+
+        self.assertEquals(
+            True, "Description: test description" in self.flow_log["info"]
+        )
+        # TODO: We should probably assert on output after description
```

Comment:
This is another good reason to make the function return a string and leave outputting it up to the caller. It makes it easier to test.